docs/developer/permissions.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Permissions
+sidebar_label: Permissions
+---
+
+## User permissions
+
+User permissions are defined by it's group. Organizing access rights into [Grups](developer/api-reference/objects/group.mdx) helps in defining staff users roles in the organization.
+
+Examples of groups:
+- Translators - `MANAGE_TRANSLATIONS` permission.
+- Customer support - `MANAGE_ORDERS` and `MANAGE_USERS` permissions.
+
+### Creating and removing groups
+- [permissionGroupCreate](developer/api-reference/mutations/permission-group-create.mdx)
+
+Request:
+```graphql
+mutation {
+  permissionGroupCreate(
+    input: {
+      addPermissions: [MANAGE_GIFT_CARD, MANAGE_DISCOUNTS]
+      addUsers: []
+      name: "Sale managers"
+    }
+  ) {
+    errors {
+      message
+    }
+    group {
+      id
+      name
+      permissions {
+        name
+      }
+    }
+  }
+}
+```
+
+Successful response:
+```json
+{
+  "data": {
+    "permissionGroupCreate": {
+      "errors": [],
+      "group": {
+        "id": "R3JvdXA6NDM=",
+        "name": "Sale managers",
+        "permissions": [
+          {
+            "name": "Manage sales and vouchers.",
+          },
+          {
+            "name": "Manage gift cards.",
+          }
+        ]
+      },
+    }
+  }
+}
+```
+
+To remove group use [permissionGroupDelete](developer/api-reference/mutations/permission-group-delete.mdx) mutation.
```

Comment:
To remove group use [permissionGroupDelete](developer/api-reference/mutations/permission-group-delete.mdx) mutation.